web/src/app/browser/src/contextManager.ts

```diff
@@ -350,13 +350,13 @@ export default class ContextManager extends ContextManagerBase<BrowserConfigurat
     // original context element may have been lost.
     this.restoreLastActiveTarget();
```

Comment:
There are still many uses of the term `Target` which I think generally means `TextStore`. These could be updated in a future PR?